examples/wifi.yml

```diff
@@ -0,0 +1,37 @@
+kernel:
```

Comment:
All the other examples can be used as is, ie you can do `moby build <example>.yml`, but you can't do it with this one. I know @deitch suggested to move it here, but I'm not in favour as this is more of a template than a fully functioning example (like the others). I would prefer if this in the documentation instead.